dll/win32/shell32/lang/en-GB.rc

```diff
@@ -64,7 +64,7 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "C&ut", IDM_CUT
```

Comment:
Why is the "u" separator not OK?

Reply:
en-US was initially introduced as &U and then later corrected to &T in the ReactOS history. The chance that en-GB uses a different accelerator than en-US.rc, and then specifically one that we accidentally used here wrongly in the beginning, is close to 0%. en-US and en-GB accelerators are the same in >= 99% of the cases.
Ftr: I gave the individual justification in all the individual changesets. It is worth to have a look at their description, I committed in babysteps to be able to give that exact level of information in there already.